### ISSUE TITLE
An ebook can get "Prerecorded Audio Only" and "Readable in read aloud or dynamic braille"

### ISSUE THREAD

**Issue by gautierchomel** [w3c/publ-a11y#651]
An ebook with "accessModeSufficient="textual" and "accessModeSufficient="auditory" will get "Prercorded Audio Only" and "Readable in read-aloud or dynamic braille", which are somewhat conflicting. This will happen with large Full daisy collections.

We should either:
* do not mention "only" (opt for something like" "Full prerecorded Audio"), or, 
* make sure that "accessModeSufficient="textual" is not present with "accessModeSufficient="auditory" to make sure the conflict does not happen (meaning that we should introduce an additional sentence for "Full prerecorded Audio"

**Comment by mattgarrish** [comment]
I think this is because of a typo in the xpath looking for media overlays:

> LET synchronised_pre_recorded_audio be the result of calling [check for node](https://w3c.github.io/publ-a11y/a11y-meta-display-guide/2.0/draft/techniques/epub-metadata/#check-for-node) on package_document, `/package/metadata/meta[@property="schema:accessibilityFeature" and normalize-space()="sychronizedAudioText"]`.

The value check is missing a "n": sy**n**chronizedAudioText

I fixed this in the viewer and it reports the correct string.

**Comment by gautierchomel** [comment]
It does not prevent the posibility of "audio only" + "braille" which is contradictory.

**Comment by mattgarrish** [comment]
Yes, when that typo is corrected then it will accurately state "Prerecorded audio synchronized with text". That's not contradictory with braille as braille compatibility is informed by the textual sufficient access mode.

**Comment by mattgarrish** [comment]
And if someone isn't putting in that an epub has text and audio synchronization, then I'd argue that's bad metadata. Audio playback relies on media overlays which always requires some minimal text and audio synchronization. If you don't have a device that supports media overlays there isn't a way to listen to the audio. EPUBs aren't audiobooks.
